metadata-ingestion/src/datahub/ingestion/transformer/extract_ownership_from_tags.py

```diff
@@ -25,32 +24,17 @@
 
 
 class ExtractOwnersFromTagsConfig(ConfigModel):
-    tag_prefix: Optional[str]
-    tag_pattern: Optional[str]
+    tag_pattern: str
```

Comment:
let's not make this required

```suggestion
    tag_pattern: str = ""
```